tests/scripts/all.sh

```diff
@@ -399,9 +420,23 @@ pre_check_git () {
     fi
 }
 
+pre_back_up () {
+    # Back up the configuration file. The cleanup function will restore it.
```

Comment:
No, that's not right. `cleanup` moves the backup file to the main file. Therefore, as soon as one cleanup is done, the backup no longer exists.